.github/workflows/continuous-integration.yml

```diff
@@ -44,3 +44,36 @@ jobs:
 
       - name: "Test fixes"
         run: "vendor/bin/phpunit --coverage-text"
+
+  coding-standards:
+    name: "Coding Standards"
+
+    runs-on: "ubuntu-18.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
```

Comment:
I copied this from another repo, but maybe I should use 7.4 here? I see no reason not to.

Reply:
Shouldn't it represent the minimum supported PHP version?

Reply:
In fact there is a very good reason: if you use the latest version, `phpcs` might ask you use recent features of the language, like it does with return type hints.